single-sign-on/index.md

```diff
@@ -71,7 +77,7 @@ The following video walks you through the process of configuring SSO.
 
 7. Proceed to **add your domain** before you test and enforce SSO.
 
-### Azure AD IdP configuration
+### Azure AD IdP configuration with Open ID Connect
 
```

Comment:
Some users may land directly on this section. Can we also add the note here to convey that users must follow these instructions only when configuring Open ID Connect with Azure AD?

Reply:
Updated